CHANGELOG.md

```diff
@@ -4,8 +4,8 @@ x.y.z Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-cocoa/issues/????), since v?.?.?)
-* None.
+
+* Fixed a crash that would occur when observing unmanaged Objects in multple views. 
```

Comment:
Typo in multple and this needs to mention that it's related to the swiftui stuff.